CHANGELOG.md

```diff
@@ -120,4 +122,4 @@ be used by projects having custom modules not in LORIS. (PR #5913)
 - The tool `phpstan` has been added to our automated test suite. (PR #4928)
 - Config files for static analysis have been moved to the `test/` directory. (PR #5871)
 - Dashboard was refactored to turn panels into module widgets. (PR #5896)
-- Add CSSGrid component type (PR #6090)
\ No newline at end of file
+- Add CSSGrid component type (PR #6090)
```

Comment:
I don't see why this is highlighted as a change, it looks like both lines are exactly the same

Reply:
I don't know what changed but copy-pasting the line seems to propose a change.

Reply:
it's impossible to commit the suggestion though, it says `this suggestion is invalid because no changes were made to the code`